mysql-test/main/outfile_fifo.test

```diff
@@ -0,0 +1,44 @@
+--source include/not_windows.inc
+--echo #
+--echo # MDEV-32061: Allow INTO OUTFILE to writo to a fifo
+--echo # This test (as well as the feature itself) does not currently
+--echo # work on Windows, although one day it probably could.
+--echo #
+CREATE TABLE t1 (val1 VARCHAR(255), val2 VARCHAR(255));
+INSERT INTO t1 VALUES('hello', 'there');
+SELECT * FROM t1;
+SELECT * INTO OUTFILE '../../tmp/fifo' FROM t1;
+TRUNCATE TABLE t1;
+
+# Let's make sure that `cat` which was started by the .sh script has
+# had a chance to complete.
+
+--disable_result_log
+--disable_query_log
+let $counter= 1000;
```

Comment:
```suggestion
let $counter= 600;
while ($counter)
{
--error 0,1
--file_exists $MYSQL_TMP_DIR/done.txt
  if ($errno)
  {
    dec $counter;
    real_sleep 0.1;
  }
  if (!$errno)
  {
    let $counter= 0;
  }
}
--file_exists $MYSQL_TMP_DIR/done.txt
```
The last file_exists will error if its not there. Otherwise use mysqltest bits to ensure the file exists.